src/main/groovy/com/brightsparklabs/gradle/baseline/BaselinePlugin.groovy

```diff
@@ -351,45 +358,139 @@ public class BaselinePlugin implements Plugin<Project> {
 
         // Error early if configuration in invalid.
         if (bucketNameIsEmpty) {
-            def error = "`bslGradle.deploy.s3.bucketName` cannot be null or empty. Value was: `${bucketName}`"
+            def error = "`bslGradle.deploy.s3.bucketName` cannot be null or empty. Value was:" +
+                    " `${bucketName}`"
```

Comment:
It annoying how IntelliJ does this for groovy. Revert it.

Presumably we're using spotless to format this anyway, so we don't strictly need IntelliJ jumping in. 


You might be able to prevent by setting the following:

IntelliJ -> Settings -> Editor -> Code Style -> Groovy -> Hard wrap at: 200


Then letting spotless clean up things later.